test/Target/SubstraitPB/Import/literal.textpb

```diff
@@ -10,6 +10,60 @@
 # RUN:   --split-input-file="# ""-----" --output-split-marker="// -----" \
 # RUN: | FileCheck %s
 
+# CHECK:      substrait.plan version 0 : 42 : 1 {
+# CHECK-NEXT:   relation
+# CHECK:         %[[V0:.*]] = named_table
+# CHECK-NEXT:    %[[V1:.*]] = project %[[V0]] : tuple<si1> -> tuple<si1, !substrait.uuid> {
+# CHECK-NEXT:    ^[[BB0:.*]](%[[ARG0:.*]]: tuple<si1>):
+# CHECK-NEXT:      %[[V2:.*]] = literal "c48ffa9e-64f4-44cb-ae47-152b4e60e77b" : !substrait.uuid
+# CHECK-NEXT:      yield %[[V2]] : !substrait.uuid
+# CHECK-NEXT:    }
+# CHECK-NEXT:    yield %[[V1]] : tuple<si1, !substrait.uuid> 
+
+relations {
+  rel {
+    project {
+      common {
+        direct {
+        }
+      }
+      input {
+        read {
+          common {
+            direct {
+            }
+          }
+          base_schema {
+            names: "a"
+            struct {
+              types {
+                bool {
+                  nullability: NULLABILITY_REQUIRED
+                }
+              }
+              nullability: NULLABILITY_REQUIRED
+            }
+          }
+          named_table {
+            names: "t1"
+          }
+        }
+      }
+      expressions {
+        literal {
+          uuid: "c48ffa9e-64f4-44cb-ae47-152b4e60e77b"
```

Comment:
I don't think that this is how the Substrait spec intends to format the UUID value...

Reply:
I am pretty sure that the `substrait-validator` interprets the literals in protobuf plans directly as 16-byte values, not as hex formatted values. See [parsing](https://github.com/substrait-io/substrait-validator/blob/68ba684d/rs/src/parse/expressions/literals.rs#L757-L781) logic.